### ISSUE TITLE
Outstanding Pull Requests

### ISSUE THREAD

**Issue by mikedupuis** [AllAlgorithms/c#30]
There are 9 outstanding pull requests. Can they please be reviewed and merged in a timely manner?

**Comment by abranhe** [comment]
@mikedupuis thank for being interested in our project, we are really sorry for this, but we are a small team reviewing a bunch of **PR**s, but they will be for sure reviewed!

**Comment by christianbender** [comment]
@abranhe Can I help you with processing the pull requests? At the moment I support [this project](https://github.com/TheAlgorithms)

**Comment by abranhe** [comment]
@christianbender Yes it would be great have some support reviewing PRs, I haven't been active a lot lately on this project, so any help would be great.

**Comment by christianbender** [comment]
@abranhe Yes I'm comfortable with C/C++.

**Comment by abranhe** [comment]
I'll add you rn as a contributor, again **Thanks**

**Comment by christianbender** [comment]
@abranhe No, problem.

**Comment by christianbender** [comment]
@abranhe Thanks

**Comment by abranhe** [comment]
@christianbender I already added you as a contributor

I am also working on creating real libraries for this project, so far I have started:

- [Javascript](https://github.com/abranhe/allalgorithms-js)
- [Python](https://github.com/abranhe/allalgorithms-python)
- [Java](https://github.com/abranhe/allalgorithms-java)

I am still working on the C and C++ started kid for those libraries.

**Comment by christianbender** [comment]
@abranhe Sounds good. I'm also competent in Python, JavaScript and Java
